geonode/thumbs/tests/test_integration.py

```diff
@@ -702,3 +703,29 @@ def test_map_custom_thumbs(self):
             self.map_composition.refresh_from_db()
 
             self._fetch_thumb_and_compare(self.map_composition.thumbnail_url, expected_thumb)
+
+    @on_ogc_backend(geoserver.BACKEND_PACKAGE)
+    @timeout_decorator.timeout(LOCAL_TIMEOUT)
+    @override_settings(
+        THUMBNAIL_BACKGROUND={
+            "class": "geonode.thumbs.background.WikiMediaTileBackground",
+        }
+    )
+    def test_UTM_dataset_thumbnail(self):
+        res = None
+        try:
+            dt_files = [os.path.join(os.path.abspath(os.path.dirname(__file__)), 'data', 'WY_USNG.zip')]
+            defaults = {"owner": GeoNodeThumbnailsIntegration.user_admin}
+            # raises an exception if resource_type is not provided
+            GeoNodeThumbnailsIntegration.rm.ingest(dt_files)
```

Comment:
Everything declared in the `setUpClass` is available with `self`, so I suggest replacing `GeoNodeThumbnailsIntegration` with `self` even to make it more readable. In this way looks like an external object is handling the information

Reply:
Done